.github/workflows/RunTests.yml

```diff
@@ -21,8 +21,8 @@ on:
   pull_request:
   workflow_dispatch:
   schedule:
-    # Run the job every 4 hours
-    - cron:  '0 */4 * * *'
+    # Run the job every 24 hours at midnight UTC
```

Comment:
We discussed this previously, and thought that we're getting value of frequent runs. Lets keep it 4 for now ?